valhalla/meili/match_result.h

```diff
@@ -94,7 +94,7 @@ struct EdgeSegment {
 
 struct MatchResults {
   MatchResults(std::vector<MatchResult>&& results, std::vector<EdgeSegment>&& segments, float score)
-      : results(results), segments(segments), score(score) {
+      : results(std::move(results)), segments(std::move(segments)), score(score) {
```

Comment:
maybe using std::forward to handle rv, and std::move to handle lv ref? Just some nitpicking